# Experiment 2 (windmill) specific parameters

# Bounding boxes (in initialization frame, relative to PlayerStart)
map_bounding_volume:
  x_min: -14
  x_max: 26
  y_min: -20
  y_max: 20
  z_min: -7
  z_max: 13

target_bounding_volume:
  x_min: 2.5
  x_max: 9.5
  y_min: -7
  y_max: 7
  z_min: -2
  z_max: 13
  
# System Constraints
planner:
  system_constraints:
    v_max: 1.0
    a_max: 1.0
    yaw_rate_max: 1.57
